tests/flow/test_rdbs.py

```diff
@@ -69,6 +70,15 @@ def testRDBCompatibility():
         newDbFileName = r.execute_command('config', 'get', 'dbfilename')[1].decode('ascii')
         env.assertEqual(newDbFileName, dbFileName)
         OLD_KEYS.sort()
+
+        if(fileName == "1.4.9_with_avg_ctx.rdb"):
+            keys = r.keys()
+            keys.sort()
+            assert keys == [b'ts1', b'ts2']
+            assert r.execute_command('ts.info', 'ts1') == [b'totalSamples', 2, b'memoryUsage', 4240, b'firstTimestamp', 100, b'lastTimestamp', 120, b'retentionTime', 0, b'chunkCount', 1, b'chunkSize', 4096, b'chunkType', b'compressed', b'duplicatePolicy', None, b'labels', [], b'sourceKey', None, b'rules', [[b'ts2', 1000, b'AVG']]]
```

Comment:
what do we test here? that the key loaded correctly? how do we verify the context is also correct?